requirements.txt

```diff
@@ -454,7 +454,7 @@ urllib3==1.26.18
     #   requests
     #   responses
     #   sentry-sdk
-validators==0.20.0
+validators
```

Comment:
How come `validators` is not pinned to a version? This seems a bit strange considering the other packages all are.

Reply:
Fixed